src/main/java/org/teamresistance/frc/Drive.java

```diff
@@ -0,0 +1,38 @@
+package org.teamresistance.frc;
+
+import org.strongback.components.Stoppable;
+import org.strongback.components.ui.ContinuousRange;
+import org.strongback.drive.MecanumDrive;
+
+/**
+ * @author Shreya Ravi
+ */
+public class Drive implements Stoppable {
+
```

Comment:
Nitpick (sorry, I should have brought this up earlier): I prefer not to have a blank line after the class signature if the next line is an instance variable.